.travis.yml

```
@@ -4,17 +4,25 @@ language: python
 python: 3.8
 cache: pip
 before_install: pip install --upgrade pip setuptools six
-install: pip install black flake8
+install: pip install black flake8 mypy pytest-cov
```

```suggestion
install: pip install black flake8
```
Let's preserve the fast fail.

---

This will fail the `Project Euler` build always as there is no `pytest-cov` in it, we're installing it only on the `Build` job.
Another solution would be to include the `install` part in both jobs or install it in `before_script` part. Can we do the latter thing?